l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -662,7 +662,23 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
         bytes32 _assetId,
         address _depositSender,
         bytes calldata _data
-    ) external payable override onlyAssetRouter onlyL1 {}
+    ) external payable override onlyAssetRouter onlyL1 {
+        delete settlementLayer[_chainId];
+
+        IStateTransitionManager(stateTransitionManager[_chainId]).bridgeClaimFailedBurn({
```

Comment:
if you did not change the code of the STM it still looks like this:

```
    /// @notice Called by the bridgehub during the failed migration of a chain.
    /// @param _chainId the chainId of the chain
    /// @param _assetInfo the assetInfo of the chain
    /// @param _prevMsgSender the previous message sender
    /// @param _data the data of the migration
    function bridgeClaimFailedBurn(
        uint256 _chainId,
        bytes32 _assetInfo,
        address _prevMsgSender,
        bytes calldata _data
    ) external {
        // todo
    }
``` 

please replace the todo with a comment on why this method is empty